src/plugins/organisation/PMRWindow/CMakeLists.txt

```diff
@@ -24,14 +19,7 @@ ADD_PLUGIN(PMRWindow
         src/pmrwindowwindow.ui
     INCLUDE_DIRS
         src
-    DEFINITIONS
-        ${LIBGIT2_DEFINITIONS}
-        ${ZLIB_DEFINITIONS}
     PLUGINS
         Core
-        ${LIBGIT2_PLUGIN}
-        ${ZLIB_PLUGIN}
-    PLUGIN_BINARIES
-        ${LIBGIT2_PLUGIN_BINARY}
-        ${ZLIB_PLUGIN_BINARY}
```

Comment:
You need to reference the libgit2 and zlib plugins otherwise, on OS X, the ```PMR``` window won't be able to find them (check with ```otool -L```).